modules/connectivity-mapping-using-the-roxctl-netpol-connectivity-map-command.adoc

```diff
@@ -11,60 +12,18 @@ include::snippets/technology-preview.adoc[]
 Connectivity mapping provide insights into the allowed connections between different workloads based on network policies defined in Kubernetes manifests.
 You can visualize and understand how different workloads in your Kubernetes environment are allowed to communicate with each other according to the network policies you set up.
```

Comment:
After this paragraph, missing a general intro of the command itself : `roxctl netpol connectivity map <folder-path>`
(similar to the description of the `roxctl netpol generate` command and of the `roxctl netpol connectivity diff`